src/stylus/fablevision/fv-gene-label.styl

```diff
@@ -0,0 +1,40 @@
+.fv-layout
+
+  .geniblocks.fv-gene-label
+    font-family: 'Yanone Kaffeesatz'
+    font-size: 1.5rem
+    color: white
+    position: absolute
+    display: flex
+    flex-direction: row
+    justify-content: flex-end
+
+    &.m
```

Comment:
If the stripes are placed on the chromosomes for artistic reasons (e.g. to space them out in an artistically pleasing manner), then the labels should be placed relative to the art rather than according to the BioLogica species definition. If the stripes are meant to be biologically accurate, then they and their labels should be placed dynamically according to the BioLogica species definition. My inclination would be to do the latter for flexibility as well as accuracy, but ultimately that's a pedagogical determination rather than an engineering one. In the former case, with artistically placed stripes, specifying the positions of the labels in CSS may well be appropriate, as long as the JavaScript to set the appropriate classes is straightforward.